soc/wch/ch32v/qingke_v2a/Kconfig.soc.ch32v006

```diff
@@ -3,7 +3,7 @@
 
 config SOC_CH32V006
 	bool
-	select SOC_SERIES_CH32V00X
+	select SOC_SERIES_QINGKE_V2A
```

Comment:
The original is correct. Also, the CH32V00x series is a QINGKE_V2C.

Reply:
fixed